samples/doc_writer_sample_output.py

```diff
@@ -560,7 +572,7 @@
                                         "title": ""
                                     }
                                 ],
-                                "returns": "vocab:singleClass",
+                                "returns": "https://hydrus.com/api/singleClass",
```

Comment:
Same for returns, it would be vocab:singleClass. We need the definition, there is no definition on https://hydrus.com/api/singleClass, that is for the instance